compiler/span/src/symbol.rs

```diff
@@ -254,6 +262,7 @@ symbols! {
     increment,
     inline,
     input,
+    interpret,
```

Comment:
Some other options to consider are `evaluate`, `run`, `script`? Also fine with `interpret` just want to make sure we consider all options before adding to the language.

`script` does feel like it is a concept at a higher level. Like `program` vs `script`.

Any other keywords to consider?

Reply:
I vote for `script` as well given the nature of these functions. `script` also fits nicely as a nounc along with `function` and `transition`.